portable/CCS/MSP430X/portext.asm

```diff
@@ -48,7 +48,7 @@ portSAVE_CONTEXT .macro
 
     ;Save the remaining registers.
     pushm_x #12, r15
-    mov.w   &usCriticalNesting, r14
+    mov_x   &usCriticalNesting, r14
```

Comment:
Both these changes look suspicious as after the update the two changed lines exchange data between a 20-bit register and a 16-bit variable.  How has this been tested?

Reply:
Thank you for the review, we have ordered a board for the validation part. I will move the PR to draft till then.